assets/onboarding/use-onboarding-api.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Use Onboarding REST API for the given step.
+ *
+ * Loads data via the GET method for the endpoint, and provides a submit function that sends data to the endpoint
+ * via POST request.
+ *
+ * @param {string} step Onboarding step endpoint name.
+ * @return {OnboardingApiHandle} handle
+ *
+ * @typedef {Object} OnboardingApiHandle
+ * @property {boolean} isBusy Loading state.
+ * @property {Object} data API response from GET call to endpoint.
+ * @property {function(Object)} submit Method to POST to endpoint.
+ */
+export function useOnboardingApi( step ) {
+	const [ data, setData ] = useState( {} );
+	const [ isBusy, setBusy ] = useState( false );
+	let path = `/sensei/v1/onboarding/${ step }`;
+	useEffect( () => {
+		fetchData();
```

Comment:
I added a `useCallback` in 9e5561a. This implementation might not be very long-lived as the recommended way to do all this would be through a `wp-data` store, instead of directly using `apiFetch`, so we should probably move to that at some point.